docs/changes.rst

```diff
@@ -1,6 +1,13 @@
 Changes
 =======
 
+Version 1.6.7
```

Comment:
Next version is [1.6.6](https://github.com/petl-developers/petl/releases)

Reply:
I have made the changes. And yes I will need the feature soon, my time is using PETL heavily.